adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -15,18 +15,18 @@ import 'package:flutter/material.dart';
 /// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
 enum AdaptiveWindowType {
   xs,
-  s,
-  m,
-  l,
+  sm,
```

Comment:
Probably shouldn't be part of this change, but you should consider using the full words that are in the material guidance. As in `small`, `medium`.

This would fit better with the dart style guide of avoiding abbreviations unless the abbreviations are more common than the word itself: https://dart.dev/guides/language/effective-dart/design#avoid-abbreviations

Reply:
Done